.travis.yml

```diff
@@ -1,43 +1,47 @@
 ---
 language: node_js
 node_js:
-  - "5.10.0"
+  # we recommend testing addons with the same minimum supported node version as Ember CLI
+  # so that your addon works for all apps
+  - "4"
```

Comment:
I think we need to changes this version either to `6` or `7`, so it matches the one specified on the _package.json_